createNativeStackNavigator/README.md

```diff
@@ -50,7 +50,7 @@ String that applies `rtl` or `ltr` form to the stack. On Android, you have to ad
 
 Whether you can use gestures to dismiss this screen. Defaults to `true`,
 
-Gestures are only supported on iOS. They can be disabled only when `stackPresentation` is `push`.
+Gestures are only supported on iOS.
```

Comment:
I changed this line, since, from iOS 13, this prop will also disable the possibility to dismiss a modal with a swipe gesture.